mb-solr/src/main/resources/oxml.xml

```diff
@@ -288,6 +171,9 @@
         <java-type name="EventList">
             <java-attributes>
                 <xml-element java-attribute="event" name="events"/>
+                <xml-element java-attribute="event" name="events">
+                    <xml-java-type-adapter value="org.musicbrainz.search.solrwriter.moxy.EventAdapter"/>
+                </xml-element>
             </java-attributes>
         </java-type>
```

Comment:
This isn’t new to this PR but it looks like one of these attributes is unneeded and should probably be removed later on.